src/data/board.py

```diff
@@ -91,6 +91,14 @@ def no_result(self) -> bool:
     def result_str(self) -> str:
         return str(self.result)
 
+    @property
+    def last_result_update(self) -> float | None:
+        return self.stored_board.last_result_update
+
+    @property
+    def last_result_update_str(self) -> str:
+        return format_timestamp(self.stored_board.last_result_update) if self.stored_board.last_result_update else ''
```

Comment:
```suggestion
        return format_timestamp(self.last_result_update) if self.last_result_update else ''
```
calls to the stored objects are only necessary for setting them (self.stored_board.last_result_update = None), otherwise that's why the property is there